src/python/grpcio/grpc/__init__.py

```diff
@@ -649,6 +649,10 @@ def unary_unary(
 
     Args:
       method: The name of the RPC method.
+      request_serializer: Optional method for serializing the request
+      message. Request goes unserialized in case None is passed.
```

Comment:
Indent another two spaces when continuing a parameter description to a new line; it should be easy to count the number of arguments to the behavior by counting the number of lines indented merely two spaces relative to "Args:".